sound/soc/sof/amd/acp-loader.c

```diff
@@ -83,11 +83,21 @@ int acp_dsp_block_write(struct snd_sof_dev *sdev, enum snd_sof_fw_blk_type blk_t
 		}
 		dest = adata->data_buf + offset;
 		adata->fw_data_bin_size = size + offset;
+		adata->is_dram_in_use = true;
```

Comment:
I would split this patch in two with first the DRAM usage (no functionality change) and then the SRAM parts.

Reply:
will split the patch series